docs/releases/changelog.md

```diff
@@ -11,9 +11,37 @@ It pulls in the latest version of upstream libc++ and marks the beginning of
 
 !-->
 
+## libcu++ 1.6.0 (CUDA Toolkit 11.5)
+
+libcu++ 1.6.0 is a major release. It adds two new functions to the `cuda::std::barrier` API and
+uses `<nv/target>` as the primary dispatch mechanism for `cuda::std::atomic`. This is an ABI breaking release.
```

Comment:
Go look up the changelog entry for the last new ABI, and make sure we are using the same wording. I think the summary for the last ABI change explained that the new one was now the default, and that the old one was still supported.